tools/nl/querygen/README.md

```diff
@@ -0,0 +1,17 @@
+# Query Generation
```

Comment:
nit: It would be great to know why this is needed? :) As in, why are we using seeds for rewriting by PaLM?

Reply:
Good point, added!